### ISSUE TITLE
NGram draft model returns probabilities as logits

### ISSUE THREAD

**Issue by paudelnirajan** [michaelpginn/spec-decoding#29]
- `predict` method in `ngram.py` returns probabilities not logits. and we are doing `log_softmax` later in spec_decode. 
- should we be passing `logits` instead of `probabilities`?
